### DISCUSSION TITLE
The Ability to use Vectorized Text (Labels)

### DISCUSSION THREAD

**Discussion by MangoDragonHub** [godotengine/godot-proposals#5050] [GUI]
It would be nice to have the ability to have the option if you want to use standard fonts both in a rasterized or vectorized way. With Screen monitors ranging in different sizes and games running in different resolutions, I think the user should have the ability to chose what format those text elements so the resolution of those text elements are losseless rather pixelated.

A good example of why this should be a feature is with Material Maker, Zooming in or out of each node that I'm working with degrades and pixelizes all the text elements. It gets difficult to tell which node I'm reading at unless I zoom in at a certain size, so it would be nice if they stay crisp and easiily visible no matter the size or zoom the text is.

**Comment by theraot** [comment]
Godot 4.0.

**Comment by Calinou** [comment]
This is available in 4.0 with MSDF fonts (which can be toggled in the Import dock, or by enabling it on the default project font thanks to https://github.com/godotengine/godot/pull/60513). However, this won't be backported to Godot 3.x.
